strr-examiner-web/app/components/ApplicationInfoHeader.vue

```diff
@@ -131,7 +131,11 @@ const registrationCountdown = computed(() => {
         <strong>{{ t('strr.label.submitted') }}</strong>
         {{ dateToString(activeHeader.applicationDateTime, 'y-MM-dd a', true) }}
         ({{ dayCountdown(activeHeader.applicationDateTime.toString(), true) }} days ago)
-        <template v-if="activeHeader.nocEndDate">
+        <template v-if="activeHeader.status === ApplicationStatus.DECLINED">
+          | <strong>{{ t('strr.label.declinedDate') }}</strong>
+          {{ dateToString(activeHeader.decisionDate, 'y-MM-dd', true) }}
```

Comment:
There is no `declinedDate` in the response, so using `decisionDate`, assuming this is the last updated date for terminal states.